docs/containers.conf.5.md

```diff
@@ -268,6 +268,9 @@ Path to the directory where CNI configuration files are located.
 ## ENGINE TABLE
 The `engine` table contains configuration options used to set up container engines such as Podman and Buildah.
 
+**build_image_format**="oci"
+Default image format for container engine builders. (OCI or Docker)
```

Comment:
Your example has "oci", but your valid values are "OCI" and "Docker".  Does casing matter?  I'd suggest:
```suggestion
The default image format for container engine builders. Valid values are "oci" (default) or "docker".
```

Reply:
Then based on the above, change the below comments as applicable.

Reply:
Fixed